### DISCUSSION TITLE
Degen Blockchain airdrop tips

### DISCUSSION THREAD

**Discussion by soheltaj7** [blockscout/blockscout#9973] [Q&A] [answered]
![IMG_20240429_115443](https://github.com/blockscout/blockscout/assets/36280079/0e6b1a9d-552f-4e5c-af81-55f38ec200ee)
![IMG_20240429_115411](https://github.com/blockscout/blockscout/assets/36280079/89ff8767-114a-4fd9-84a2-b445feafe603)

**

> `### This Degen chain Tips Airdrop Real or fack
![IMG_20240429_115443](https://github.com/blockscout/blockscout/assets/36280079/13cde2bb-1bcd-42ed-aa1b-ace8268fe15e)
![IMG_20240429_115411](https://github.com/blockscout/blockscout/assets/36280079/6cb13de2-81c7-423d-b10f-65c5fa1a2306)
`

**

Airdrop 2
In Season 3, you can earn points on Farcaster by receiving tips from others. You'll have a daily tip allowance, visible on this page once you connect your wallet. To leave a tip, comment under a cast with "amount $DEGEN", for example, "420 $DEGEN". If you exceed your daily limit, additional tips won't count. You can tip on multiple casts each day. Tips in specific channels get a boost. Control who receives Degen – happy tipping!

**Comment by ekawibisono** [comment, marked as answer]
Hello @soheltaj7 This is blockscout Github, please contact Degen Team.
